src/tusb_option.h

```diff
@@ -278,8 +278,14 @@
     #error there is no benefit enable hub with max device is 1. Please disable hub or increase CFG_TUSB_HOST_DEVICE_MAX
   #endif
 
-  #ifndef CFG_TUH_ENUMERATION_BUFSZIE
-    #define CFG_TUH_ENUMERATION_BUFSZIE 256
+  // Backwards compatible misspelling
+  #ifdef CFG_TUH_ENUMERATION_BUFSZIE
+    #define CFG_TUH_ENUMERATION_BUFSIZE CFG_TUH_ENUMERATION_BUFSZIE
+    #warning "CFG_TUH_ENUMERATION_BUFSZIE is misspelled, please use CFG_TUH_ENUMERATION_BUFSIZE"
+  #endif
```

Comment:
I'm not sure if you care about such backwards compatibility, since TinyUSB ist still at v0.x. I can remove it if you want.

Reply:
no need for backward compatible with host at the moment, since it is not used much. Please just replace the typo